background.js

```diff
@@ -1,8 +1,2714 @@
 (function () {
-    const URL_REGEX = /^(https?:\/\/docs\.python\.org\/)([23][^\/]*?)(\/.*)/;
     const SPECIAL_CASES = {
-        "/library/sets.html" : "/library/stdtypes.html#set",
-        "/library/stringio.html": "/library/io.html#io.StringIO"
+        // TODO: these commented links might break when 2to3 is removed
```

Comment:
Maybe now it would make sense to move all of those `SPECIAL_CASES` to a separate file? `background.js` has become very large because of that.